### PR TITLE
[REVIEW][FEATURE][UWANG-160] - Create Sidebar Component

### PR DESCRIPTION
**Pull request by maulanafikriash**
#### Description

> Description of features, improvements design.

#### Ticket Reference(s)

[UWANG-160](https://bluehabit.atlassian.net/browse/UWANG-160)

##### UI Changes

ssside-make
mini-make

##### Coverage

> Test coverage screenshot.

N/A

#### Note(s)

> Mention other user or extra note unrelated to the MR here.
@triandamai 

[UWANG-160]: https://bluehabit.atlassian.net/browse/UWANG-160?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by triandamai — CHANGES_REQUESTED** [review]
cek ya @maulanafikriash

File: components/Sidebar.vue [hunk 1]
```diff
@@ -0,0 +1,95 @@
+<script lang="ts">
+export default {
```
[the remark this answers is not in the document]
Comment (maulanafikriash) [reply components/Sidebar.vue]: Sip udah ku ganti mas @triandamai

[the remark this answers is not in the document]
Comment (triandamai) [reply components/Sidebar.vue]: conflict @maulanafikriash

[the remark this answers is not in the document]
Comment (maulanafikriash) [reply components/Sidebar.vue]: oke udah ku fix mas @triandamai

**Review by triandamai — APPROVED** [review]

**Pull request merged**